python/cudf/cudf/_lib/string_casting.pyx

```diff
@@ -613,38 +592,22 @@ def htoi(Column input_col, **kwargs):
     -------
     A Column of integers parsed from hexadecimal string values.
     """
-
-    cdef column_view input_column_view = input_col.view()
-    cdef type_id tid = <type_id> (
-        <underlying_type_t_type_id> (
-            SUPPORTED_NUMPY_TO_LIBCUDF_TYPES[cudf.dtype("int64")]
-        )
+    plc_column = plc.strings.convert.convert_integers.hex_to_integers(
+        input_col.to_pylibcudf(mode="read"),
+        dtype_to_pylibcudf_type(cudf.dtype("int64"))
     )
```

Comment:
Since this is a hardcoded cuDF dtype, would it make sense to instead use the corresponding pylibcudf datatype here and skip the conversion altogether?


```suggestion
        plc.DataType(plc.TypeId.INT64)
```